1.1.0: using a logging config causes exception

I've been using Borg 1.0.x with a [simple logging config](https://github.com/borgbackup/borg/files/1369192/logging.conf.txt) to get a logging behaviour suitable for cronjobs, that is: everything goes to the logfile, errors and warnings also go to stderr (and are sent via mail, by cron). Using the same logging config with Borg 1.1.0 causes an exception:

```
2017-10-09 06:05:09 [ERROR] Local Exception
2017-10-09 06:05:09 [ERROR] Traceback (most recent call last):
  File "borg/archiver.py", line 4024, in main
  File "borg/archiver.py", line 3952, in run
  File "borg/archiver.py", line 130, in wrapper
  File "borg/remote.py", line 562, in __init__
  File "borg/remote.py", line 699, in call
  File "borg/remote.py", line 841, in call_many
  File "borg/remote.py", line 989, in handle_remote_line
AttributeError: 'Logger' object has no attribute 'json'
```

When not using a logging config, `setup_logging` will set up two loggers, the second one explicitly named `borg` and having a custom `json` attribute (see logging.py, lines 97-99). While I could add a `borg` logger to my logging config there seems to be no way to add the required custom `json` attribute within the fileConfig format.

--- a/src/borg/logger.py
+++ b/src/borg/logger.py
@@ -80,6 +80,8 @@ def setup_logging(stream=None, conf_fname=None, env_var='BORG_LOGGING_CONF', lev
                 logging.config.fileConfig(f)
             configured = True
             logger = logging.getLogger(__name__)
+            borg_logger = logging.getLogger('borg')
+            borg_logger.json = json
             logger.debug('using logging configuration read from "{0}"'.format(conf_fname))
             warnings.showwarning = _log_warning
             return None

